### PR TITLE
Add support for vertical spacing in InputBarButtonItem

### PR DESCRIPTION
**Pull request by marcetcheverry**
This adds support for adding vertical padding spacing to an `InputBarButtonItem`. `UIStackView` supports items with different heights.

The use case for this is a simple bar with multiple `InputBarButtonItem`s that are based on just text. This allows you to easily add some vertical padding.

### CODE REVIEW

File: InputBarAccessoryView.podspec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Pod::Spec.new do |s|
     s.homepage = "https://github.com/nathantannar4/InputBarAccessoryView"
 
     # 6 - Source
-    s.source = { :git => "https://github.com/nathantannar4/InputBarAccessoryView.git", :tag => "#{s.version}"}
+    s.source = { :git => "https://github.com/marcetcheverry/InputBarAccessoryView.git", :branch => "swift5-size-padding" }
```
Comment (nathantannar4) [inline InputBarAccessoryView.podspec]: Can you please undo this change?

File: InputBarAccessoryView/Controls/InputBarButtonItem.swift [hunk 2]
```diff
@@ -29,30 +29,30 @@ import UIKit
 
 /**
  A InputItem that inherits from UIButton
- 
+
```
Comment (nathantannar4) [inline InputBarAccessoryView/Controls/InputBarButtonItem.swift]: Looks like you have added some spaces here, could you please clean that up? I use Xcode's automatically trim trailing whitespace setting. (for lining purposes)

File: InputBarAccessoryView.podspec (hunk 1 above)
Comment (marcetcheverry) [reply InputBarAccessoryView.podspec]: Will do, sorry about this, needed it for testing.

File: InputBarAccessoryView/Controls/InputBarButtonItem.swift (hunk 2 above)
Comment (marcetcheverry) [reply InputBarAccessoryView/Controls/InputBarButtonItem.swift]: This was probably introduced by a file wide Xcode `CTRL-i`, I also have that setting. I usually ignore whitespace on commits so it slipped through.

**Comment by marcetcheverry** [comment]
I've made the code style changes and I've added the:
```setContentHuggingPriority(UILayoutPriority(rawValue: 1000), for: .vertical)```

> Also, do you think there is a use case where someone would need vertical and horizontal spacing? Maybe we need flexibleVertical and flexibleHorizontal?

Indeed, that could be useful, but for my purposes I just needed to add some vertical padding to a horizontally arranged set of `InputBarButtonItem`s, I assume this can also be done by adjusting the height constant of the bar that contains it, but I could not find an easy way to do that in `InputBarAccessoryView`.

Either way I think this is a good addition, and in the future someone can submit a PR to add those vertical flexible spacings if they really need a vertical toolbar.

**Comment by marcetcheverry** [comment]
Was this merged?

**Comment by nathantannar4** [comment]
Oh, no I was doing some cleanup and deleted the `swift5`branch since its already merged with `master` and it looks like that auto closed this

**Comment by nathantannar4** [comment]
I can't seem to re-open this, can you please create a new branch targeting `master`
